### PR TITLE
Updated Turkish Localization

### PR DESCRIPTION
**Pull request by NovaDiablox**
Added Crosshair Fixes and Additions Text

### CODE REVIEW

File: lua/terrortown/lang/tr.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@
 local L = LANG.CreateLanguage("tr")
 
 -- Compatibility language name that might be removed soon.
--- the alias name is based on the original TTT language name
```
Comment (TimGoll) [inline lua/terrortown/lang/tr.lua:6]: is there a reason you removed that line?

File: lua/terrortown/lang/tr.lua [hunk 2]
```diff
@@ -820,17 +820,17 @@ L.aw_tod2_text = "raundun başlamasından hemen sonra öldü."
 
 -- New and modified pieces of text are placed below this point, marked with the
 -- version in which they were added, to make updating translations easier.
-
+-- New comment format for this language file is YY-MM-DD
```
Comment (TimGoll) [inline lua/terrortown/lang/tr.lua:823]: no, we use YYYY-MM-DD. Our language tool will revert your comment changes anyway

**Review by TimGoll — CHANGES_REQUESTED** [review]

File: lua/terrortown/lang/tr.lua (hunk 1 above)
Comment (NovaDiablox) [reply lua/terrortown/lang/tr.lua:6]: When I first started translating the file, the Turkish language did not exist in the original TTT. I translated the original later. I can add it back if it is neccesary.

File: lua/terrortown/lang/tr.lua (hunk 2 above)
Comment (NovaDiablox) [reply lua/terrortown/lang/tr.lua:823]: Ok no problem 👌

Comment (TimGoll) [reply lua/terrortown/lang/tr.lua:823]: you can keep it that way and I run the language tool on your PR. So you don't have to manually edit it

File: lua/terrortown/lang/tr.lua (hunk 1 above)
Comment (TimGoll) [reply lua/terrortown/lang/tr.lua:6]: yes, please keep it there

**Comment by NovaDiablox** [comment]
> Fine by me, once @TimGoll ran the language tool.
> 
> we really need the CI for that :D and a guide for others to use it

Yes I would like to learn it. By the way I think we should contact other translators, as new texts strings are added they will pile up.

**Comment by TimGoll** [comment]
> > Fine by me, once @TimGoll ran the language tool.
> > we really need the CI for that :D and a guide for others to use it
> 
> Yes I would like to learn it. By the way I think we should contact other translators, as new texts strings are added they will pile up.

If you want to run this yourself, clone this repository on your machine: https://github.com/TTT-2/ttt2-language_parser, navigate to the folder of the language parser and then run the command you find in the readme.

**Comment by NovaDiablox** [comment]
> > > Fine by me, once @TimGoll ran the language tool.
> > > we really need the CI for that :D and a guide for others to use it
> > 
> > Yes I would like to learn it. By the way I think we should contact other translators, as new texts strings are added they will pile up.
> 
> If you want to run this yourself, clone this repository on your machine: https://github.com/TTT-2/ttt2-language_parser, navigate to the folder of the language parser and then run the command you find in the readme. 

I'm done with the parsing.

**Review by TimGoll — APPROVED** [review]
looks good to me, thank you so much!

**Pull request merged**